distribution/docs/src/main/resources/_contents/_catalog-contents/extending-catalog-contents.adoc

```diff
@@ -3683,7 +3684,7 @@ Once the REST Endpoint receives the GeoJSON Metacard, it is converted to a Catal
 
 .Example HTTP POST of a local `metacard.json` file using the Curl Command
 ----
-curl -X POST -i -H "Content-Type: application/json" -d "@metacard.json" http://localhost:8181/services/catalog
+curl -X POST -i -H "Content-Type: application/json" -d "${at-symbol}metacard.json" ${public_url}/services/catalog
```

Comment:
Should we use the secure_url for all these examples? I noticed we do in running-contents.adoc